app/javascript/modules/datadog.js

```diff
@@ -0,0 +1,20 @@
+import { datadogLogs } from '@datadog/browser-logs'
+
+const initDatadog = () => {
+    const ddEnv = window.location.hostname === 'www.citizensadvice.org.uk' ? "prod" : "dev";
```

Comment:
Does this app have review environments? I'm wondering if it would be better to get the environment from the DOM (and ultimately an environment variable) rather than hardcoding the prod URL like this.

Reply:
Yep I could make it an env var like the version number.  I hadn't thought to upload the files for review apps, at the moment it would only run on successful deployment to `qa` and `prod`